test/modules/test_nf4_linear.py

```diff
@@ -170,5 +164,18 @@ def test_load_from_nf4_diff_meta(self):
         assert other_mod.param.block_size == 64
         assert other_mod.param.scaler_block_size == 1
 
+    def test_to_copy(self):
+        inpt_tensor = torch.rand(128, device='cpu')
```

Comment:
need to initialize it as bf16?

Reply:
I specifically wanted to try some other dtypes. To be rigorous I'd try them all, but that might not be worth the additional test power unless we do something special.